packages/flutter/lib/src/cupertino/segmented_control.dart

```diff
@@ -62,7 +63,7 @@ const Duration _kFadeDuration = Duration(milliseconds: 165);
 /// See also:
 ///
 ///  * <https://developer.apple.com/design/human-interface-guidelines/ios/controls/segmented-controls/>
-class SegmentedControl<T> extends StatefulWidget {
+class CupertinoSegmentedControl<T> extends StatefulWidget {
```

Comment:
I know it's a new class, and probably doesn't have a lot of usage yet, but you should follow the [Breaking Changes](https://flutter.io/design-principles/#handling-breaking-changes) process.  This is a breaking change in the API.

You might want to separate out the rename from the other changes, and do it in a later PR, since it might be after you leave when it's finally approved (because of the waiting period on breaking changes).